src/po-helpers.js

```diff
@@ -22,11 +22,23 @@ export function buildPotData(translations) {
 }
 
 
-export function applyReference(poEntry, node, filepath) {
+export function applyReference(poEntry, node, filepath, location) {
```

Comment:
I think it would be great to have simple unit test for this function in tests/unit/